packages/terra-consumer-icon/scripts/src/generate-icon/Icon.js

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
yes. Actually I was feeling lazy to put on all the copied files so I thought I will point out the lines where I made changes. I will do it now.